src/ResourceManager/Resources/Commands.Resources.Test/ScenarioTests/DeploymentTests.cs

```diff
@@ -44,6 +44,12 @@ public void TestNestedDeploymentFromTemplateFile()
         }
 
         [Fact]
+        public void TestCrossResourceGroupDeploymentFromTemplateFile()
```

Comment:
@luanshixia this test failed the on-demand job. Please make this test a check-in test so it is run on the regular job.